reccmp/isledecomp/formats/pe.py

```diff
@@ -995,3 +1028,33 @@ def read(self, vaddr: int, size: int) -> Optional[bytes]:
         # Reading off the end will most likely misrepresent the virtual addressing.
         _size = min(size, section.size_of_raw_data - offset)
         return bytes(section.view[offset : offset + _size])
+
+    def read_initialized(
```

Comment:
This looks like it is a bigger change, but it's rather making explicit what was implicit earlier. Previously, if you got an unexpected `None` in `read()`, you would also run into an exception. I went over all uses of `read()` and replaced the ones that would have crashed on a `None` response.